clang/lib/3C/ProgramInfo.cpp

```diff
@@ -332,19 +332,24 @@ bool ProgramInfo::link() {
     // everything about it.
     // Some global symbols we don't need to constrain to wild, like
     // malloc and free. Check those here and skip if we find them.
-    if (!G->hasBody() && !isExternOkay(FuncName)) {
+    if (!llvm::is_contained(DefinedExternalFunction, FuncName) &&
+        !isExternOkay(FuncName)) {
```

Comment:
Reiterating earlier question: Is `DefinedExternalFunction` really needed? Why is what Kyle figured out before insufficient?